lean4/src/putnam_2022_b1.lean

```diff
@@ -6,12 +6,12 @@ open Polynomial
 theorem putnam_2022_b1
 (n : ℕ)
 (P : Polynomial ℝ)
-(B : Polynomial ℝ)
+(B : ℕ → ℝ)
```

Comment:
nit:
```suggestion
(b : ℕ → ℝ)
```
to match the english. I think the uppercase `B` was chosen to be the polynomial with coefficients `b`, but of course that turned otu to be silly.

Reply:
Removed this from this PR. Can approve and merge #227.